src/com/superiornetworks/icarus/commands/Command_gcmd.java

```diff
@@ -0,0 +1,117 @@
+package com.superiornetworks.icarus.commands;
+
+import com.superiornetworks.icarus.ICM_Rank;
+import com.superiornetworks.icarus.ICM_Utils;
+import net.pravian.bukkitlib.command.BukkitCommand;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandParameters(name="gcmd",description="Make someone run a command!",usage="/gcmd <player> <command>",rank=ICM_Rank.Rank.SENIOR)
+public class Command_gcmd extends BukkitCommand {
+    
+    @Override
+    public boolean run(CommandSender sender, Command cmnd, String string, String[] args)
+    {
+    
+        Player p = (Player) sender;
```

Comment:
This will cause a NPE (when sent from console) if you don't check for console first.